workflow/controller/steps.go

```diff
@@ -68,9 +68,9 @@ func (woc *wfOperationCtx) executeSteps(ctx context.Context, nodeName string, tm
 		{
```

Comment:
I can't comment on unchanged lines, but I think the `defer` above this can also be modified to use the same `node`, similar to the DAG code

Reply:
fixed